### ISSUE TITLE
Nodemon restarts but doesn't apply changes

### ISSUE THREAD

**Issue by Mattperkinsee** [remy/nodemon#1985]
- Versions: 
- `nodemon -v`: 2.0.15
- Operating system/terminal environment (powershell, gitshell, etc): Windows 10 (Ubuntu SBS 18.04.2 LTS (Bionic Beaver))
- Command you ran: nodemon --exec babel-node -- index.js

### Expected behaviour
App to restart with updated changes

### Actual behaviour
App restarts but does not apply any of the updated changes.

### Steps to reproduce
Install nodemon 2.0.15, spin up a simple node.js app with express

---

**Comment by remy** [comment]
Can't replicate with given information. Please provide pared down code - otherwise I suspect this will languish and auto close.

**Comment by Mattperkinsee** [comment]
I will work on getting an example. After writing it, I suspect it might have to do something with babel. I actually forgot that I was using that in this project.

**Comment by pavbyte** [comment]
is it like updating the res.send with 'hello world' to 'hello Marr' and after saving the code, nodemon restarts but the changes is not shown on the browser on localhost??

**Comment by pavbyte** [comment]
> I will work on getting an example. After writing it, I suspect it might have to do something with babel. I actually forgot that I was using that in this project.

is it like updating the res.send with 'hello world' to 'hello Marr' and after saving the code, nodemon restarts but the changes is not shown on the browser on localhost??

**Comment by cybrown** [comment]
I've managed to replicate this issue on Windows 10, using this repository https://github.com/cybrown/nodemon-bug-http, nodejs version 16.14.0, npm version 8.3.1 and gitbash.
It works in cmd and powershell though.
It works in gitbash when the web server is not listening.

**Comment by eferrabelo1114** [comment]
Having the same issue right now 

**nodejs version 16.15.0
npm version 8.1.3
and nodemon 2.0.16**

Did the exact same thing, created a simple express app with a simple endpoint that gets an array of users. When I update the users list, I see 
![image](https://user-images.githubusercontent.com/74121414/167755939-86e65dfe-ef0e-4991-b927-4b4613d07f7d.png)

but when I refresh the page nothing changes.

I've tried installing nodemon globally, changing PATH variables, using VSC terminal(Powershell and Git Bash), and using Windows Powershell. I've tried using nodemon versions 2.0.7 and 2.0.14. 

Only thing I haven't tried is a different version of nodejs
